R/osmdata.R

```diff
@@ -104,28 +104,37 @@ get_osmdata <- function(
   )
 
   osm_data <- list(
+    bb = bb,
     boundary = boundary,
     river_centerline = river$centerline,
     river_surface = river$surface
   )
 
   # Retrieve streets and railways based on the aoi
   if (!is.null(network_buffer)) {
-    aoi <- get_river_aoi(river, bb, buffer_distance = network_buffer)
-    osm_data <- append(osm_data, list(aoi = aoi))
+    aoi_network <- get_river_aoi(river$centerline, bb,
```

Comment:
For consistency, I thought it would be good to actually add the AoI used to retrieve buildings as well (if needed). Hence the renaming to be more specific on this being only for the network.